modules/nw-ne-openshift-dns.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+// * assembly/networking
+
+[id='networking-ne-openshift-dns-{context}']
+= {product-title} DNS
+
+If you are running multiple services, such as frontend and backend services for
+use with multiple pods, environment variables are created for user names,
+service IPs, and more so the frontend pods can communicate with the backend
+services. If the service is deleted and recreated, a new IP address can be
+assigned to the service, and requires the frontend pods to be recreated to pick
+up the updated values for the service IP environment variable. Additionally, the
+backend service has to be created before any of the frontend pods to ensure that
+the service IP is generated properly, and that it can be provided to the
+frontend pods as an environment variable.
+
+For this reason, {product-title} has a built-in DNS so that the services can be
+reached by the service DNS as well as the service IP/port. the client to
```

Comment:
Something funky going on with the end of this - maybe the last part is supposed to be deleted or turned into a full sentence?